frontend/views/containers/access/LoginForm.vue

```diff
@@ -124,7 +124,13 @@ export default ({
         await finishedLoggingIn
         await this.postSubmit()
         this.$emit('submit-succeeded')
-        requestNotificationPermission()
+
+        requestNotificationPermission({
+          notificationPayload: {
+            title: 'Logged in',
+            body: 'Welcome again!'
+          }
+        })
```

Comment:
Here and below we should separate these out, along with a comment, e.g.:

```js
const granted = requestNotificationPermission() === 'granted'
// TODO: remove in production - this is just for testing the notification
if (granted) { /* post notification */ }
```

Same for the code below in `SignupForm.vue`